lib/SoftwareTree.py

```diff
@@ -1,301 +1,213 @@
 import sublime_plugin, sublime
+from copy import deepcopy
+from threading import Thread, Timer, Event 
+from .SoftwareOffline import getSessionSummaryData, launchCodeTimeMetrics
+from .SoftwareUtil import *
+from .SoftwareModels import SessionSummary
+from .SoftwareWallClock import *
+from .SoftwareFileChangeInfoSummaryData import *
+from .SoftwareRepo import *
 
-class OpenTreeView(sublime_plugin.TextCommand):
+icons = getIcons()
+tree_view = None
+orig_layout = None
+NO_ID = 'NO_ID'
+CODETIME_TREEVIEW_NAME = 'Code Time Tree View'
+shouldOpen = True
```

Comment:
Should some of these variables be stored inside of the class instance instead of globally?

Reply:
Some of these variables exist outside of the class because their state needs to persist across tree view refreshes or need to be accessed in other files.